From 5d193a4cccb619f5b66515e9465cc2188a52ed62 Mon Sep 17 00:00:00 2001
From: Andrew Senetar <arsenetar@voltaicideas.net>
Date: Wed, 25 Aug 2021 21:50:07 -0500
Subject: [PATCH] Patch for cocoa

---
 core/app.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/app.py b/core/app.py
index 37921e94..02f45416 100644
--- a/core/app.py
+++ b/core/app.py
@@ -562,7 +562,7 @@ class DupeGuru(Broadcaster):
         self.ignore_list_dialog.refresh()
         p = op.join(self.appdata, "exclude_list.xml")
         self.exclude_list.load_from_xml(p)
-        self.exclude_list_dialog.refresh()
+        #self.exclude_list_dialog.refresh()
 
     def load_directories(self, filepath):
         # Clear out previous entries
-- 
2.30.1 (Apple Git-130)

